build-system/tasks/ava.js

```diff
@@ -15,29 +15,30 @@
  */
 'use strict';
 
-const gulp = require('gulp');
-const gulpAva = require('gulp-ava');
+const {exec} = require('../common/exec');
 
 /**
  * Runs ava tests.
- * @return {!Vinyl}
+ * @param {function} cb
  */
-function ava() {
-  return gulp
-    .src([
-      require.resolve('./get-zindex/test.js'),
-      require.resolve('./prepend-global/test.js'),
-    ])
-    .pipe(
-      gulpAva({
-        'concurrency': 5,
-        'failFast': true,
-      })
-    );
+function ava(cb) {
+  const avaCli = 'node_modules/ava/cli.js';
+  const testFiles = [
+    require.resolve('./get-zindex/get-zindex.test.js'),
```

Comment:
is there a similar mechanism to require.resolve if we were to switch over to es modules in our build system?

Reply:
We could use `path.resolve`, but it will need an additional `require('path')` and maybe some logging changes. I was going to do a wholesale refactor of `build-system` to modernize it at some point. Okay to leave this change for a follow up?

Reply:
yep OK to leave it. just making sure we have an analogue mechanism when we decide to switch over

Reply:
Changed this so it doesn't use `path.resolve` or `require.resolve`.